lib/onc_certification_g10_test_kit/all_resources.rb

```diff
@@ -0,0 +1,75 @@
+require_relative 'g10_options'
+
+module ONCCertificationG10TestKit
+  module AllResources
+    # This list a list of all resource types mapped to USCDI data classes or elements,
```

Comment:
Stick module documentation above the module.

```
module ONCCertificationG10TestKit
  # This is a list...
  module AllResources
```

Also, a typo here: This *list* a list of all